tasks/1.txt

```diff
@@ -8,3 +8,7 @@ Create a SQL query to return next data ordered by city and then by name:
 | Employy Id | Employee Full Name | Title | City |
 
 [Employee Full Name] - First Name and Last Name.
+
+db.employees.aggregate(
+    [{ $project: { EmployeeID: 1,"Employee Full Name": { $concat: [ "$FirstName", " ", "$LastName" ]}, Title: 1, City: { $ifNull: [ "$City", "-" ]}}},
+       { $sort: { City: 1, "Employee Full Name": 1}}]);
```

Comment:
You can use sort before project with some changes.
Also please include an index to optimize sort